src/Parsers/ASTSystemQuery.h

```diff
@@ -33,6 +33,7 @@ class ASTSystemQuery : public IAST, public ASTQueryWithOnCluster
 #endif
         DROP_FILESYSTEM_CACHE,
         DROP_SCHEMA_CACHE,
+        DROP_SCHEMA_FORMAT_CACHE,
```

Comment:
And since this query has extra arguments, it need custom formatter, here is an example https://github.com/ClickHouse/ClickHouse/pull/55277

Reply:
Thanks! Fixed.